tests/e2e/voice/calls/test_update_call.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from sinch.domains.voice.models.calls.responses import UpdateVoiceCallResponse
+
+
+@pytest.mark.skip(reason="Conference endpoints have to be implemented first.")
```

Comment:
I think it was one of the purpose of this PR to implement the conference endpoints. Please check is this annotation needs to be removed

Reply:
yeah; I was working on those tests but forgot to finalise fixtures for them. I'll fix it now